packages/mrm-task-readme/index.spec.js

```diff
@@ -37,3 +37,28 @@ it('should add a readme', async () => {
 		omitBy(vol.toJSON(), (v, k) => k.startsWith(__dirname))
 	).toMatchSnapshot();
 });
+
+it('should add a readme with custom file name', async () => {
+	vol.fromJSON({
+		[`${__dirname}/templates/Readme.md`]: fs
+			.readFileSync(path.join(__dirname, 'templates/Readme.md'))
+			.toString(),
+		'/package.json': stringify({
+			name: 'unicorn',
+			repository: 'gandalf/unicorn',
+		}),
+	});
+
+	task(
+		await getTaskOptions(task, false, {
+			name: 'Gandalf',
+			url: 'https://middleearth.com',
+			license: 'MIT',
+			licenseFile: "${('license-' + license).toUpperCase()}.md",
+		})
+	);
+
+	expect(
```

Comment:
If you only care about the correct file name here, I'd explicitly test that, and avoid a snapshot.

Reply:
Ok, I've switched to a regex test.